src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. A good fallback for Roboto would be `Arial, sans-serif`. So the line would look like this: `font-family: Roboto, Arial, sans-serif;`